### ISSUE TITLE
SemesterSeeder creates the semester twice

### ISSUE THREAD

**Issue by kdmnk** [luksan47/mars#430]
The issue can be that the database updates slowly, and the cycle is too fast.
![image](https://user-images.githubusercontent.com/56202545/95653049-e4fd4180-0af5-11eb-99e4-f8ee58575c0b.png)
Also, we should overwrite the collection's getter with something like this:
```
 ->sortByDesc(function ($semester, $key) {
                return $semester->getStartDate();
            })
```
to sort semesters automatically.

**Comment by kdmnk** [comment]
In #426 I changed updateOrCreate to getOrCreate, now only the current semester is shown twice.

**Comment by luksan47** [comment]
Hm, I'm not sure why this is happening. Might be something with pivots.

**Comment by luksan47** [comment]
This should be fixed now with #426.
